inc/saihostintf.h

```diff
@@ -556,6 +556,9 @@ typedef enum _sai_hostif_packet_attr
     /** Ingress LAG [sai_object_id_t] (for receive-only) */
     SAI_HOSTIF_PACKET_INGRESS_LAG,
 
+    /** Ingress VLAN [sai_vlan_id_t] (for receive-only) */
+    SAI_HOSTIF_PACKET_INGRESS_VLAN,
```

Comment:
That's VLAN pkt is forwarded over ASIC. It may be either VLAN pkt is received with or in case pkt is not tagged - PVID.